### PR TITLE
Creates a Windows `How-to` on Building

### PR DESCRIPTION
**Pull request by JeffNeff**
Includes everything except for the `How to install the binary` bit. As I'm unclear on that step myself lol. I just found this repository/project today, so I have not had a ton of time to play yet.

Closes issue #256

### CODE REVIEW

**Comment by LeeAlephium** [comment]
Thank you @JeffNeff ! This is good but needs a few minor adjustments...!

We need to keep in mind the audience that this document is for:

* Windows users
* Non developers

I think it's fair to expect Windows web developers to understand how to build this project from scratch using git and node.

Due to this, the document can be more focused if you remove the parts about git, contributing, and a text editor.

Essentially just focus on getting the user the zip file of a release, building that, and installing it.

FWIW the final binary is in `./dist`. It's convenient to suggest they can place this on their Desktop so let's go for this until we have a proper installation process.

Again thank you very much for the work! I don't want to discourage you in any way, you're doing great :heart:

**Comment by JeffNeff** [comment]
Ah okay got it! I understand the user story a bit better now. The initial Issue asked specifically for git, so I assumed a mixed dev/user audience. Let me refine this a bit!

**Comment by LeeAlephium** [comment]
My bad, I should've been a bit clearer on that :laughing: Thank you for the understanding

**Comment by JeffNeff** [comment]
No problems!! Ill have to do it tonight or tomorrow morning tho because "work" and things xd.

File: docs/building/windows.md [hunk 1]
```diff
@@ -4,69 +4,46 @@ This document is written with the intention of demonstrating the build process i
...
 
 ### Clone the Project
 
-Begin by cloning or downloading the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet). This can be accomplished by selecting the green `<>Code` button and then electing either `Open with Github Desktop` or `Download ZIP`.
+Download the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet).
 
-**Note**: If you plan to make changes, its a good idea to go ahead and [create a new branch](https://www.softwaretestinghelp.com/github-desktop-tutorial/) before starting.
+This can be accomplished by selecting the green `<>Code` button and then `Download ZIP` from the dropdown.
+
+Or by following this [link](https://github.com/alephium/desktop-wallet/archive/refs/heads/master.zip).
```
Comment (LeeAlephium) [inline docs/building/windows.md:25]: Target latest release, not master

File: docs/building/windows.md [hunk 2]
```diff
@@ -4,69 +4,46 @@ This document is written with the intention of demonstrating the build process i
...
 
-If VSCode is NOT avalible. One can open a new terminal within the local repo with the following steps:
+2. Navigate to, and open, the repo folder.
 
-1. Open `File Explorer`
-    
-2. Navigate to, and open, the repo folder. 
-    
-3. In the navigation bar type the following:
+3. In the navigation bar (where the file path is currently located) type the following:
```
Comment (LeeAlephium) [inline docs/building/windows.md:36]: Could you include a screenshot for this?

File: docs/building/windows.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Building
+
+
+### Clone the Project
+
+Download the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet).
+
+This can be accomplished by selecting the green `<>Code` button and then `Download ZIP` from the dropdown.
```
Comment (LeeAlephium) [inline docs/building/windows.md:23]: A screenshot here would be good too. There will be many people unfamiliar with GitHub.

File: docs/building/windows.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Or by following this [link](https://github.com/alephium/desktop-wallet/archive/refs/heads/master.zip).
+
+
+### Navigate to the Project Within a Terminal
+
+Open a new terminal within the local repo with the following steps:
+
+1. Open `File Explorer`
+
+2. Navigate to, and open, the repo folder.
```
Comment (LeeAlephium) [inline docs/building/windows.md:34]: For what it's worth I recall you can directly go into zip files on Windows. Will everything still "just work" or should they first extract the zip?

Comment (JeffNeff) [reply docs/building/windows.md:34]: You are correct! I should include the step to extract the files!

File: docs/building/windows.md (hunk 2 above)
Comment (JeffNeff) [reply docs/building/windows.md:36]: Yea no problem!

File: docs/building/windows.md (hunk 1 above)
Comment (JeffNeff) [reply docs/building/windows.md:25]: got it! Good catch!

Comment (JeffNeff) [reply docs/building/windows.md:25]: There is no `release` branch so if I target a specific release the documentation will need updated (or will be out of date) every release cycle.

Comment (LeeAlephium) [reply docs/building/windows.md:25]: For now use https://github.com/alephium/desktop-wallet/releases/tag/latest even if it's not present. I'll deal with this tomorrow

Comment (LeeAlephium) [reply docs/building/windows.md:25]: /cc @mvaivre @nop33

Comment (JeffNeff) [reply docs/building/windows.md:27]: got it!

Comment (nop33) [reply docs/building/windows.md]: You use this link which always points to the latest release: https://github.com/alephium/desktop-wallet/releases/latest
Using `master` is not a good idea, as the code is not always made to work with mainnet, but with a future version of node/explorer.

Comment (nop33) [reply docs/building/windows.md]: but we will have to keep your tag always up to date when we release a new release. The link I shared always points to the latest release without any additional work.

**Comment by LeeAlephium** [comment]
@JeffNeff it's not sure much where the navigation bar I wanted but showing what it should look like after typing "cmd" - it's actually something I've never seen myself before so it looks strange when reading it :slightly_smiling_face:

**Pull request closed without merging**